addons/udes_stock/models/stock_move.py

```diff
@@ -1,7 +1,9 @@
 import logging
+import math
```

Comment:
Nit: typos `wil` & `if moves a move` in the commit message

Reply:
I messed up my fixups and had to do a force push, so I've fixed this too.